src/utils/metrics.ts

```diff
@@ -46,20 +46,20 @@ export const blockHeight = new client.Gauge({
 });
 
 export const reorgDepth = new client.Gauge({
-  name: "watch_tower_reorg_depth",
-  help: "Depth of the current reorg",
+  name: "watch_tower_latest_reorg_depth",
+  help: "Depth of the last reorg",
   labelNames: ["chain_id"],
 });
 
-export const blockTime = new client.Gauge({
-  name: "watch_tower_block_time_seconds",
+export const blockProducingRate = new client.Gauge({
+  name: "watch_tower_block_producing_rate_seconds",
   help: "Time since last block",
   labelNames: ["chain_id"],
 });
```

Comment:
I'm unsure about this being labelled as a `rate`, as the measurement being taken is `scalar`. If we did want to stick to this kind of naming, it would be `watch_tower_block_production_rate_seconds`.

Even just writing this, usually when talking about a rate it's `units / sec`, where actually it could then be argued that in the case of mainnet, this metric would be expected to report 1 / 12. Per the [naming conventions](https://prometheus.io/docs/practices/naming/), with reference to `Power`, they make a statement about using a `rate()` function on a `joules` counter.

Reply:
What about just `watch_tower_block_production_seconds`?

Reply:
https://github.com/cowprotocol/watch-tower/pull/111